docs/user-guide/patterns/partial-apply.md

```diff
@@ -1,25 +1,51 @@
 (partially parametric rules)=
-Partially Parametric Rules
-==========================
+# Partially Parametric Rules
 
-````{todo}
-
-Write a function that applies a method with some arbitrary arguments and others
-passed in, and uses method otherwise.
+The provided code snippet illustrates a partially parametric rule in CVL that defines specific behavior based on the method invoked (`f`) and its arguments (`calldataargs`). Let's break down the example for better understanding:
 
 ```cvl
-function applyToUser(method f, env e, address user) {
-    if (f.selector == balanceOf(address).selector) {
-        return balanceOf(e, user);
-    }
-    if (f.selector == transfer(address,address).selector) {
-        address other;
-        return transfer(e, user, other);
-    }
-    else {
-        calldataarg arg;
-        return f(e, arg);
-    }
+rule partially_parametric_rule(env e, method f, calldataargs args)
+{
+	if (f.selector == withdraw(uint256, address).selector) {
```

Comment:
you need to use `sig:` here, also below.

Reply:
Done